### PR TITLE
Geodata parameters

### PR DESCRIPTION
**Pull request by bariela**
## Description

/geodata/:tableName route did not allow for parameters in openAPI, 
also filtering by values in parameters of geojson feature enabled

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have checked to ensure that there aren't other open [Pull Requests](../../../pulls) for the same update/change.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: api/src/openapi/apiSchema.yml [hunk 1]
```diff
@@ -295,7 +325,7 @@ paths:
         200:
           description: Successful response
 
-  /api/staticLayers:
+  /api/dataLayers:
```
Comment (srezacova) [inline api/src/openapi/apiSchema.yml:328]: Hi, we still use word static in many place in this config regarding this endpoint. Should this be renamed also?

**Comment by srezacova** [comment]
Hi, so for this to work on FE, there should be properties added also here(boxVectorSourceLoader.js LN 29):
const url = `${baseUrl}&bottomLeft=${bottomLeft.join(',')}&topRight=${topRight.join(',')}&proj=${proj}`;

**Comment by bariela** [comment]
> Hi, so for this to work on FE, there should be properties added also here(boxVectorSourceLoader.js LN 29): const url = `${baseUrl}&bottomLeft=${bottomLeft.join(',')}&topRight=${topRight.join(',')}&proj=${proj}`; but for now we won't use it?

they can be part of baseUrl (geoDataUrl)
also the api route is/can be used for dashboard, too

File: api/src/openapi/apiSchema.yml (hunk 1 above)
Comment (bariela) [reply api/src/openapi/apiSchema.yml:328]: yes, it should :) I just considered it out of scope here

File: frontend/src/ol/staticLayers/vectorSourceLoader.js [hunk 2]
```diff
@@ -18,15 +18,16 @@ const vectorSourceLoader = (layerData, handleIsLoading, title, type) => {
             let attributes = [];
 
             if (layerData.attribute) {
+              const escapedAttribute = encodeURIComponent(layerData.attribute);
```
Comment (srezacova) [inline frontend/src/ol/staticLayers/vectorSourceLoader.js:21]: This starts failing with "APIError: Parameter 'attributeId' must be url encoded. Its value may not contain reserved characters.

File: api/src/openapi/apiSchema.yml (hunk 1 above)
Comment (srezacova) [reply api/src/openapi/apiSchema.yml:328]: :) Sure. I fixed it on FE for layers to work.

**Comment by srezacova** [comment]
Hi, works right, nice job. 
Just in Mongo version for http://localhost:3000/api/geodata/slovakiaRegionsGeoData it returns:
{"type":"FeatureCollection","name":"slovakiaRegionsGeoData","features":[]}
in Postgre:
{"message":500,"stack":"APIError: select \"id\", ST_AsGeoJSON(geometry) as geometry, \"properties\" from \"slovakiaRegionsGeoData\" - relation \"slovakiaRegionsGeoData\" does not exist\n    at /app/dist/src/config/express.js:110:26\n  ...

**Pull request merged**